src/publisher/ajson_ingestor.py

```diff
@@ -163,7 +159,10 @@ def _ingest(data, force=False) -> models.ArticleVersion:
         # identity (this data already exists) and validity (this data is malformed)
 
         # validation and hash check of article-json occurs here
-        fragments.set_article_json(av, quiet=False if settings.VALIDATE_FAILS_FORCE else force, hash_check=True)
+        quiet = False if settings.VALIDATE_FAILS_FORCE else force
```

Comment:
since `settings.VALIDATE_FAILS_FORCE` is always `True` at the moment, should `quiet` be simplified to `False`?

Reply:
but maybe tests patch it to something else, hence this should stay

Reply:
> but maybe tests patch it to something else

I encountered some of that. Same with update_fragment and I didn't want to be modifying the tests for this unless I absolutely had to.